lib/everest.cpp

```diff
@@ -42,7 +42,7 @@ Everest::Everest(std::string module_id_, const Config& config_, bool validate_da
                  std::shared_ptr<MQTTAbstraction> mqtt_abstraction, const std::string& telemetry_prefix,
                  bool telemetry_enabled) :
     mqtt_abstraction(mqtt_abstraction),
-    config(std::move(config_)),
```

Comment:
moving a `const` is indeed pointless.  But it might be a better choice to make the constructor argument `Config config_` instead of const ref and dropping the move.